solr/solrj/src/java/org/apache/solr/client/solrj/io/eval/PutCacheEvaluator.java

```diff
@@ -37,18 +37,18 @@ public PutCacheEvaluator(StreamExpression expression, StreamFactory factory) thr
   }
 
   @Override
-  @SuppressWarnings({"unchecked", "rawtypes"})
   public Object doWork(Object... values) throws IOException {
-    ConcurrentMap objectCache = this.streamContext.getObjectCache();
+    ConcurrentMap<String, Object> objectCache = this.streamContext.getObjectCache();
     if(values.length == 3) {
       String space = (String)values[0];
       String key = (String)values[1];
       space = space.replace("\"", "");
       key = key.replace("\"", "");
       Object value = values[2];
-      ConcurrentMap spaceCache = (ConcurrentMap)objectCache.get(space);
+      @SuppressWarnings("unchecked")
+      ConcurrentMap<String, Object> spaceCache = (ConcurrentMap<String,Object>)objectCache.get(space);
```

Comment:
Same question as above w.r.t. `ConcurrentMap<String, Object>` vs. `ConcurrentMap<Object, Object>` type.

Reply:
I had a TODO in StreamContext that we should be able to be more specific about what objectCache contains, and went ahead and implemented that.